src/MeiliSearch/Index.php

```diff
@@ -80,7 +80,12 @@ protected function createIndex()
     {
         try {
             $this->client->createIndex($this->name, ['primaryKey' => 'id']);
-            $this->getIndex()->updateSettings($this->config['settings'] ?? []);
+
+            if (! isset($this->config['settings'])) {
+                return;
```

Comment:
Probably better to throw an exception?

```suggestion
                throw new ApiException('Could not read settings from config.);
```

Reply:
It's ok to have no settings, since Meilisearch will use the default settings in that case. 
https://docs.meilisearch.com/reference/api/settings.html#update-settings